admin/admin-static-pages.php

```diff
@@ -70,7 +71,11 @@ public function use_block_editor_for_post( $use_block_editor, $post ) {
 	 * @return void
 	 */
 	public function add_meta_boxes( $post_type, $post ) {
-		if ( 'page' === $post_type && empty( $post->post_content ) && (int) get_option( 'page_for_posts' ) === $post->ID ) {
+		if ( ! $this->should_disable_block_editor( $post ) ) {
+			return;
+		}
+
+		if ( ! use_block_editor_for_post( $post ) ) {
```

Comment:
Are we obliged to modify this method?
My plan is to mark `use_block_editor_for_post()` and `should_disable_block_editor()` as `Backward compatibility with WP < 5.8` and once we stop supporting WP < 5.8, remove these methods. So I would like to avoid to modify this method once again when this ime will come.